### ISSUE TITLE
Heroku deploy results in errors

### ISSUE THREAD

**Issue by grokify** [grokify/fastly-logging-proxy#1]
Deploying commit  0e107b7c0917bb1dcc2feb3084f4b72908d27269

results in the following error:

```
-----> Building on the Heroku-20 stack
-----> Determining which buildpack to use for this app
-----> Go app detected
-----> Fetching jq... done
-----> Fetching stdlib.sh.v8... done
-----> 
       Detected go modules via go.mod
-----> 
       Detected Module Name: github.com/grokify/fastly-logging-proxy
-----> 
-----> New Go Version, clearing old cache
-----> Installing go1.18.3
-----> Fetching go1.18.3.linux-amd64.tar.gz... 
 !     Push failed due to an unrecognized error, and we've been notified.
 !     Please try pushing again.
 !     If the problem persists, see https://help.heroku.com/ and provide Request ID 7c45c0e1-4c24-72b3-4322-c179233b306a.
```

**Comment by grokify** [comment]
This appears to be a temporary error as a redeploy is successful, but now the app results in the following error:

```html
<!DOCTYPE html>
	<html>
	  <head>
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<meta charset="utf-8">
		<title>Application Error</title>
		<style media="screen">
		  html,body,iframe {
			margin: 0;
			padding: 0;
		  }
		  html,body {
			height: 100%;
			overflow: hidden;
		  }
		  iframe {
			width: 100%;
			height: 100%;
			border: 0;
		  }
		</style>
	  </head>
	  <body>
		<iframe src="//www.herokucdn.com/error-pages/application-error.html"></iframe>
	  </body>
	</html>
```

**Comment by grokify** [comment]
Fixed in c0e9bba9d9d1dd2abd764aed496b76b6ca99b3ea and 7e546e767344a0ddefcfa9f4cc50464f0ed814d8
